README.md

```diff
@@ -48,53 +54,71 @@ yarn add @openmrs/esm-form-engine-lib@latest
 
 ### Local development
 
+#### Clone the repository
+
 ```bash
 git clone git@github.com:openmrs/openmrs-esm-form-engine-lib.git
+```
 
-# OR with HTTPS
+Or using https:
 
+```bash
 git clone https://github.com/openmrs/openmrs-esm-form-engine-lib.git
 ```
 
-#### Install Dependencies
+#### Install dependencies
 
 ```bash
 yarn
 ```
 
 #### Build the library
+
 ```bash
 yarn build
 ```
 
-#### Link the library to the dependent frontend module
+#### Link the library to the consuming frontend module
 
-To test your changes locally, you need to link the library to the dependent frontend module. Presently, this library is used by the following modules:
+To test your local Form Engine changes, you'll need to link the library to a frontend module that uses it (like Form Builder or Patient Chart). Here's how:
 
-- [openmrs-esm-form-builder](https://github.com/openmrs/openmrs-esm-form-builder)
-- [openmrs-esm-patient-chart](https://github.com/openmrs/openmrs-esm-patient-chart)
+1. Build the library
 
-To link the library to the dependent frontend module, run the following command from the root of the frontend module:
+```bash
+yarn build
+```
+
+2. Copy the relative path to the built library
+
+Copy the relative path to the built library by running the following command on Mac:
 
 ```bash
-yarn link `path/to/openmrs-esm-form-engine-lib`
+pwd | pbcopy # copy the path to form-engine-lib to the clipboard
 ```
 
-For example, if you are working on the `openmrs-esm-form-builder` module, you would run the following command:
+Or the equivalent command on Linux:
 
 ```bash
-pwd | pbcopy # copy the path to form-engine-lib to the to the clipboard
-cd `path/to/openmrs-esm-form-builder`
-yarn link `pbpaste` # paste the path to the openmrs-form-engine-lib
+pwd -W | pbcopy # copy the path to form-engine-lib to the clipboard
```

Comment:
@ibacher I seem to remember us suggesting using `xclip` at some point. Not sure if the command I've suggested actually works.

Reply:
`pwd -W` doesn't work for me on my Mac. I think `xclip` is Linux. Why were you removing the other lines here, though? `pbcopy` will just put things on the Mac clipboard (internally called "pastebin", hence `pbcopy` and `pbpaste`).

Reply:
Nvm... I see the other lines were moved.

Reply:
Windows has a `clip` command, but I can't find anything that would actually paste from the clipboard.